Sync .pyre_configuration.external files for those that got out of sync with last upgrade

connernilsen opened the pull request:
Summary:
This downgrades the .pyre_configurations in the following projects to keep their internal and external configurations synced.
- beanmachine/beanmachine/ppl
- tools/sapp
- torchrec
- torchx

The versions were chosen to match the most recent Pyre version in [pyre-check-nightly](https://pypi.org/project/pyre-check-nightly/#history).

Differential Revision: D61211101

yikaiMeta reviewed (changes requested):
Review automatically exported from Phabricator review in Meta.

connernilsen committed:
Sync .pyre_configuration.external files for those that got out of sync with last upgrade (#2294)

Summary:
Pull Request resolved: https://github.com/pytorch/torchrec/pull/2294

X-link: https://github.com/pytorch/torchx/pull/943

This downgrades the .pyre_configurations in the following projects to keep their internal and external configurations synced.
- beanmachine/beanmachine/ppl
- tools/sapp
- torchrec
- torchx

The versions were chosen to match the most recent Pyre version in [pyre-check-nightly](https://pypi.org/project/pyre-check-nightly/#history).

Differential Revision: D61211101
